data_structures/src/superblock.rs

```diff
@@ -244,16 +335,14 @@ impl SuperBlockState {
     pub fn build_superblock(
```

Comment:
We should remove the current comments of the function and update them with some description of the new arguments